lark/exceptions.py

```diff
@@ -18,7 +17,7 @@ class ConfigurationError(LarkError, ValueError):
     pass
 
 
-def assert_config(value, options, msg='Got %r, expected one of %s'):
+def assert_config(value, options: Sequence, msg='Got %r, expected one of %s'):
```

Comment:
I think `Collection` is more correct here. a `set` is ok.

Reply:
You're right.